### PR TITLE
Fix #36568 flagandcross.com

### LINKED ISSUE #36568: flagandcross.com
### Issue URL (Incorrect Blocking)
[https://flagandcross.com/meteorologist-takes-aoc-to-school-with-lesson-about-exploiting-weather-to-push-climate-change-agenda/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fflagandcross.com%2Fmeteorologist-takes-aoc-to-school-with-lesson-about-exploiting-weather-to-push-climate-change-agenda%2F)
### Comment
Username: @JonnyTrader
> Every page I 've visited  on this site is same..  Many in main article  and Comment sections COMPLETELY unusable 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Filters: | Filter unblocking search ads and self-promotion,<br>AdGuard Base
Other extensions used: | webex  hangouts malware bytes

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36568

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1,6 +1,11 @@
 !
 !######### Annoying elements (not ads)
 !
+flagandcross.com###adtoniq-msgr-bar
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/annoyances.txt]: For English filter.

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -144,6 +144,9 @@ kiss-anime.ws##script[id^="BB_SLOT_"] ~ div[id][style^="overflow: hidden; width:
 ! ##.banner
 aplayer2.me,viralizeit.net,sex3.com,softonic.com,3pornstarmovies.com,flvto.biz,govtsmartjob.com,cutearn.ca,dfporn.net,cll.press,hellporno.net,youramateurporn.com,getlink.pw,embedtvseries.com,keygames.com,uclick.in,flv2mp3.by,doodhwali.com,smutindia.com,dvdcover.com,xxxadultphoto.com,gabicoins.com,qpdownload.com,windows10portal.com,2conv.com,javcloud.com,underhentai.net,cutwin.com,hqasiananal.com,bradsknutson.com,flv2mp3.org,subscene.com,yourlink.in,iplayer.pw,qpornx.com,sofifa.com,jawcloud.co,indiansgetfucked.com,medi-bayreuth.de,oxforddictionaries.com,tube18.sex##.banner
 !
+flagandcross.com###klicked-shopify-widget
+flagandcross.com##.contextual-ad-d
```
Comment (BlazDT) [inline EnglishFilter/sections/specific.txt]: Is not hiding any left-over for me.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (Alex-302) [reply EnglishFilter/sections/specific.txt]: @BlazDT 
image

Comment (BlazDT) [reply EnglishFilter/sections/specific.txt]: That is #adtoniq-msgr-bar. When user presses on OK, it is possible he gets more ads, therefore English filter imho.

**Comment by Alex-302** [comment]
What about this?
image

**Pull request merged**